modules/channel-web/src/backend/api.ts

```diff
@@ -279,25 +278,12 @@ export default async (bp: typeof sdk, db: Database) => {
       sanitizedPayload = _.omit(payload, [...sensitive, 'sensitive'])
     }
 
-    const event = bp.IO.Event({
-      botId,
+    const message = await bp.messaging.sendIncoming(conversationId, payload, {
```

Comment:
No because we are storing messages as part of a conversation. If we convert other channels to this api, they'll need to use getRecentConversation before sending a message.